rbac/management/group/definer.py

```diff
@@ -82,20 +84,29 @@ def seed_group() -> Tuple[Group, Group]:
     return group, admin_group
 
 
-def set_system_flag_before_update(group: Group, tenant, user) -> Optional[Group]:
+def set_system_flag_before_update(group: Group, tenant, user) -> Tuple[Optional[Group], Iterable[Relationship]]:
     """Update system flag on default groups."""
+    relations = []
     if group.system:
-        group = clone_default_group_in_public_schema(group, tenant)  # type: ignore
+        group, relations = clone_default_group_in_public_schema(group, tenant)  # type: ignore
         group_flag_change_notification_handler(user, group)
-    return group
+    return group, relations
 
 
-def clone_default_group_in_public_schema(group, tenant) -> Optional[Group]:
+def clone_default_group_in_public_schema(group, tenant) -> Tuple[Optional[Group], Iterable[Relationship]]:
     """Clone the default group for a tenant into the public schema."""
-    if settings.PRINCIPAL_CLEANUP_UPDATE_ENABLED_UMB:
-        # TODO: bootstrap the tenant to get the mapping
-        # use this for uuid instead and to remove the default role binding tuple
-        group_uuid = uuid4()
+    relationships = []
+    if settings.REPLICATION_TO_RELATION_ENABLED:
+        tenant_bootstrap_service = V2TenantBootstrapService(OutboxReplicator())
+        bootstrapped_tenant = tenant_bootstrap_service.bootstrap_tenant(tenant)
+
+        relationships = tenant_bootstrap_service.default_bindings_from_mapping(
+            bootstrapped_tenant, group.admin_default
+        )
+        if group.admin_default:
+            group_uuid = bootstrapped_tenant.mapping.default_admin_group_uuid
```

Comment:
We don't allow admin_default group to be customized. So this clone method won't apply to admin_default. We can simplify the logic to treat it as False